### DISCUSSION TITLE
how to load my external javascript file?

### DISCUSSION THREAD

**Discussion by murugappanrm** [nuxt/nuxt#16387] [Questions] [answered]
Hi,

I created an external script file "myscripts.js" and placed in under assets->js folder. The content of my script is simple:
```

function displayNote() {
  console.log("DisplayNote");
}

```
In the nuxt.config.ts included the following code:
```

/ https://v3.nuxtjs.org/api/configuration/nuxt.config
export default defineNuxtConfig({
  app: {
    head: {
      script: [
        {
          src: "~assets/js/myscripts.js",
          body: true,
        },
      ],
    },
  },
});

```
When i run yarn dev i get this error in the developer tools console:
`
Uncaught SyntaxError: expected expression, got '<'
`
Is there something wrong in my codes?

**Comment by TheAlexLichter** [comment, marked as answer]
You cannot use `~` for `useHead` calls. Instead, you rather want to put them in the `public` folder and reference them as `/js/myscripts.js`.

**Reply by murugappanrm** [reply]
I did exactly as you said and it gives the same error. Actually, it loads "myscripts.js" but it says there is a syntax error in the code which is seems odd as the code is simple and fundamental. Is there a bug in Nuxt 3 RC 13? Is there a proven working sample. The documentation is patchy on this part.

**Reply by TheAlexLichter** [reply]
Please provide a [reproduction](https://v3.nuxtjs.org/community/reporting-bugs/#create-a-minimal-reproduction) and show what you tried then...

**Reply by murugappanrm** [reply]
ok, Here is the link https://stackblitz.com/edit/github-qdj2uv?file=app.vue

**Reply by TheAlexLichter** [reply]
Works exactly as expected - 
![grafik](https://user-images.githubusercontent.com/640208/201858043-993d5384-b3c3-45bb-809d-16732430f927.png)

**Reply by murugappanrm** [reply]
Yes, it works in stackblitz. When i tested in my local pc under Windows 10, the same error occurs:
```

The script from “http://localhost:3000/js/myscript.js” was loaded even though its MIME type (“text/html”) is not a valid JavaScript MIME type. localhost:3000
Uncaught SyntaxError: expected expression, got '<' myscript.js:1
```

Then i went on to do further tests and i found these:
(1) I get this error when i test with a freshly created (today 15th November 2022) instance of nuxt 3.
(2) When i test the same js script under an instance i created a week ago, it works fine.
(3) Last few days we have been reporting issues with rc13 and since then its not been good. Refer https://github.com/nuxt/framework/discussions/8939.

I am still doing further diagnostic tests. Will update you next. After testing more, i can confirm this problem is being cause by updated version of RC 13.

**Reply by TheAlexLichter** [reply]
![grafik](https://user-images.githubusercontent.com/640208/201926481-cdcceb82-dce3-4302-9ffa-7c3385a5ad4e.png)

Works fine locally for me
